app/services/session_encryptor.rb

```diff
@@ -1,23 +1,25 @@
 class SessionEncryptor
-  def user_access_key
+  def initialize
     @user_access_key ||= begin
       key = Figaro.env.session_encryption_key
-      user_access_key = UserAccessKey.new(password: key, salt: key)
-      random_r = OpenSSL::Digest::SHA256.digest(key)
-      user_access_key.unlock(random_r)
-      user_access_key
+      UserAccessKey.new(password: key, salt: key)
     end
   end
 
+  def user_access_key
```

Comment:
should this method be called `duped_user_access_key`?

Reply:
Yep, looks like I forgot the critical `cmd+s` step.

Reply:
Fixed